### DISCUSSION TITLE
Uploading and saving Adapter-Fusion layers

### DISCUSSION THREAD

**Discussion by Soumyadeepsar** [adapter-hub/adapters#747] [Q&A] [answered]
We can upload adapters using push_adapter_to_hub() in our huggingface hub. What is the way of uploading the Adapter fusion layer (fused layer) after training on a task? So for adapter-fusion, we create a fuse block and also add a classification head. Can you tell me how to upload these layers so that they can be used later on by other people?

**Comment by calpt** [comment, marked as answer]
Hey, this functionality will be added with this PR: https://github.com/adapter-hub/adapters/pull/771

**Reply by Soumyadeepsar** [reply]
Ok, thank you for your reply. That's great news!!
